api/pkg/service/resource/resource.go

```diff
@@ -380,6 +418,25 @@ func matchesName(name string) func(db *gorm.DB) *gorm.DB {
 	}
 }
 
+func queryByName(name string, exact bool) func(db *gorm.DB) *gorm.DB {
+	if name == "" {
+		return noop
+	}
+	if exact {
+		return filterByName(name)
+	}
+	return matchesName(name)
+}
+
+// This function lowercase the first letter
```

Comment:
lowercase the first letter? or the whole string?

Reply:
It lowers the whole string, will update the comment